packages/cli/src/create/command.ts

```diff
@@ -76,7 +110,7 @@ export async function twCreate(pPath: string = "", options: any) {
     if (typeof res.projectType === "string") {
       projectType = res.projectType.trim();
     }
-  } else if (!projectType && options.template) {
+  } else if (!projectType || options.template || options.solana) {
```

Comment:
what about `options.evm`?

I'd prefer us to just infer the projectType above where we check for the flags and then not need to check again here